### DISCUSSION TITLE
[Datetime picker] Dayjs adapter

### DISCUSSION THREAD

**Discussion by seerj30** [ng-matero/extensions#146] [Q&A]
Hi all,

Is there an adapter for `dayjs` library?

For Material Datepicker I use `@tabuckner/material-dayjs-adapter`, but in combination with **MtxDatetimepickerModule** I get the following errors:
![image](https://user-images.githubusercontent.com/33357726/200555792-4d4a72cf-c862-4aa8-b407-4c2e6133f446.png)

**Comment by nzbin** [comment]
I'm sorry, the extensions adapters are based on Material. Maybe I will add dayjs adapter after Material does.
